src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="workflow?.descriptorType === 'WDL' || 'CWL'" class="panel panel-default">
```

Comment:
Yikes... Hadn't noticed this; we're displaying an empty launch with for Nextflow because of this. :(